src/Stratis.Bitcoin.Features.Miner/MiningFeature.cs

```diff
@@ -148,15 +137,15 @@ public override void Initialize()
         {
             if (this.minerSettings.Mine)
             {
-                string minto = this.minerSettings.MineAddress;
-                // if (string.IsNullOrEmpty(minto)) ;
+                string mineToAddress = this.minerSettings.MineAddress;
```

Comment:
This code should eventually go away as its not secure to pass the password on the command line (we should use swagger api or command cli instead)